lib/LedgerSMB/Upgrade_Tests.pm

```diff
@@ -122,6 +119,20 @@ C<text> is the textual value to be presented in the UI.
 
 has selectable_values => (is => 'ro', isa => 'HashRef', required => 0);
 
+sub _displayed_key {
+    my ($self,@keys) = @_;
+    for my $key (@keys) {
+        die "'$key' not displayed in test '$self->{name}'"
+            if not grep( $key, $self->{display_cols});
+    }
+};
+
+after 'selectable_values' => sub {
+    my $self = shift;
+    $self->_displayed_key(keys %{$self->{selectable_values}})
```

Comment:
Do we need to run this check on *every access* to `selectable_values`? or are you actually only trying to check it when being set?

Reply:
It should be in the builder, not on every access.